css/bootstrap5/privatebin.css

```
@@ -32,6 +32,10 @@
 	word-wrap: break-word;
 }
 
+#plaintext > a {
```

Same DRY suggestion here, merge with the rule above on 31:
```suggestion
#pastelink > a, #plaintext > a {
```

---

```suggestion
#pastelink > a, #plaintext  a,  {
```

I made a mistake…